### ISSUE TITLE
Magento upgrade 2.4.0 to 2.4.1 - Login As Customer acl.xml title attribute missing

### ISSUE THREAD

**Issue by DiegoSana** [magento/magento2#30680]
### Preconditions (*)

1. Magento 2.4.0
2. LoginAsCustomer disabled
`'Magento_LoginAsCustomer' => 0,`
`'Magento_LoginAsCustomerAdminUi' => 0,`
`'Magento_LoginAsCustomerApi' => 0,`
`'Magento_LoginAsCustomerFrontendUi' => 0,`
`'Magento_LoginAsCustomerLog' => 0,`
`'Magento_LoginAsCustomerLogging' => 0,`
`'Magento_LoginAsCustomerPageCache' => 0,`
`'Magento_LoginAsCustomerQuote' => 0,`
`'Magento_LoginAsCustomerSales' => 0,`
`'Magento_LoginAsCustomerWebsiteRestriction' => 0,`

### Steps to reproduce (*)

1. Upgrade to Magento 2.4.1;
2. Try to access the admin.

### Expected result (*)

1. Acces to the admin dashboard with your user and password

### Actual result (*)

1. Blank page with this error
```
> Exception #0 (LogicException): Could not create an acl object: Invalid Document 
Element 'resource': The attribute 'title' is required but missing.
Line: 13
```

![Screenshot from 2020-11-06 16-11-11](https://user-images.githubusercontent.com/51679138/98514811-bb5d4480-2272-11eb-8f46-fcd754e909cc.png)

2. Only Magento_LoginAsCustomerAssistance enabled
`'Magento_LoginAsCustomer' => 0,`
`'Magento_LoginAsCustomerAdminUi' => 0,`
`'Magento_LoginAsCustomerApi' => 0,`
`'Magento_LoginAsCustomerAssistance' => 1,`
`'Magento_LoginAsCustomerFrontendUi' => 0,`
`'Magento_LoginAsCustomerLog' => 0,`
`'Magento_LoginAsCustomerLogging' => 0,`
`'Magento_LoginAsCustomerPageCache' => 0,`
`'Magento_LoginAsCustomerQuote' => 0,`
`'Magento_LoginAsCustomerSales' => 0,`
`'Magento_LoginAsCustomerWebsiteRestriction' => 0,`
3. When magento try yo merge all acl.xml the resource Magento_LoginAsCustomer::login stay without title attribute

### Workarround

Enable Magento_LoginAsCustomerAdminUi
or
Enable  Magento_LoginAsCustomerLog
or
Enable all Magento_LoginAsCustomer*
or
Disable Magento_LoginAsCustomerAssistance

- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._

**Comment by DiegoSana** [comment]
This PR fix it #30681
Waiting for approval

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-38975` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by gabrieldagama** [comment]
Hi @DiegoSana. Thank you for your report. 
The issue has been fixed in magento/magento2#30681 by @DiegoSana in 2.4-develop branch
Related commit(s):
 - [381e2ea41f05c6ba581e4a0966d37eb767d3411a](https://github.com/magento/magento2/commit/381e2ea41f05c6ba581e4a0966d37eb767d3411a)
 - [09c125acd875b284a222d8cd21c6d4bf4e3b6242](https://github.com/magento/magento2/commit/09c125acd875b284a222d8cd21c6d4bf4e3b6242)
 - [a97f6c9e29f914c47c3ad21688e63659c1e45670](https://github.com/magento/magento2/commit/a97f6c9e29f914c47c3ad21688e63659c1e45670)

The fix will be available with the upcoming 2.4.2 release.
